src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -208,29 +213,33 @@ class LazyMobXTableStore<T> {
     @computed public get downloadData()
```

Comment:
yeah the table component should be completely agnostic about the form of the data, download should take the rowData and the table shouldn't have any opinion on whether its an array or not.